pkg/resource/google/google_compute_global_forwarding_rule_test.go

```diff
@@ -17,6 +18,8 @@ func TestAcc_Google_ComputeGlobalForwardingRule(t *testing.T) {
 		},
 		Checks: []acceptance.AccCheck{
 			{
+				// New resources are not visible immediately through GCP API after an apply operation.
+				ShouldRetry: acceptance.LinearBackoff(10 * time.Minute),
```

Comment:
is that 10 minutes? quite long isn't it? from your experience it makes sense?

Reply:
We could probably switch it to 5min, but it's not that important because if in real life if it took less we'll get out of this without waiting 10 mins.
It's more like if we can do more we can do less in that case.

Reply:
cool sounds great 👍